pywinauto/linux/atspi_objects.py

```diff
@@ -986,3 +1079,53 @@ def get_minimum_increment(self, g_error_pointer=None):
     @g_error_handler
     def set_current_value(self, new_value, g_error_pointer=None):
         return self._set_current_value(self._pointer, new_value, g_error_pointer)
+
+
+class AtspiDocument(object):
+
+    """Access to ATSPI Document Interface"""
+
+    _get_locale = IATSPI().get_iface_func("atspi_document_get_locale")
+    _get_locale.argtypes = [POINTER(_AtspiDocument), POINTER(POINTER(_GError))]
+    _get_locale.restype = c_char_p
+
+    _get_attribute_value = IATSPI().get_iface_func("atspi_document_get_document_attribute_value")
+    _get_attribute_value.argtypes = [POINTER(_AtspiDocument), c_char_p, POINTER(POINTER(_GError))]
+    _get_attribute_value.restype = c_char_p
+
+    _get_attributes = IATSPI().get_iface_func("atspi_document_get_document_attributes")
+    _get_attributes.argtypes = [POINTER(_AtspiDocument), POINTER(POINTER(_GError))]
+    _get_attributes.restype = c_void_p
+
+    def __init__(self, pointer):
+        """Init the ATSPI Document Interface"""
+        self._pointer = pointer
+
+    @g_error_handler
+    def get_locale(self, g_error_pointer=None):
+        """
+        Gets the locale associated with the document's content, e.g. the locale for LOCALE_TYPE_MESSAGES.
```

Comment:
(minor) All docstrings in pywinauto use infinitive form of a verb (without "s").

Reply:
Noted